torch/fx/experimental/symbolic_shapes.py

```diff
@@ -3409,7 +3450,7 @@ def _update_divisible(self):
     def simplify(self, expr: "sympy.Expr") -> "sympy.Expr":
         """Use known constraints and replacements to simplify the given expr
         """
-        expr = self.replace(expr)
+        expr = self.replace(expr, resolve_unbacked=True)
```

Comment:
default is `True` though? doesn't this mean all unbacked symints are replaced no matter what?

Reply:
Yeah lol